centrifuge-app/src/pages/Onboarding/BusinessInformation.tsx

```diff
@@ -41,17 +54,29 @@ const BusinessInformationInlineFeedback = ({ isError }: { isError: boolean }) =>
   return null
 }
 
-export const BusinessInformation = ({ nextStep, setUltimateBeneficialOwners }: Props) => {
+export const BusinessInformation = ({ backStep, nextStep }: Props) => {
   const { selectedAccount } = useWeb3()
   const { authToken } = useAuth()
+  const { onboardingUser, refetchOnboardingUser } = useOnboardingUser() as {
+    onboardingUser: EntityUser
+    refetchOnboardingUser: () => void
+  }
+
+  const isCompleted = useMemo(() => {
+    if (onboardingUser?.steps?.verifyBusiness.completed) {
+      return true
+    }
+
+    return false
+  }, [onboardingUser])
```

Comment:
```suggestion
  const isCompleted = !!onboardingUser?.steps?.verifyBusiness.completed
```
Don't think useMemo is really necessary here

Reply:
👍